morpheus/_lib/include/morpheus/messages/control.hpp

```diff
@@ -170,15 +171,15 @@ class PythonByteContainer : public std::vector<uint8_t>
 {
   public:
     PythonByteContainer() = default;
-    PythonByteContainer(pybind11::object py_obj) : m_py_obj(std::move(py_obj)) {}
+    PythonByteContainer(mrc::pymrc::PyHolder py_obj) : m_py_obj(std::move(py_obj)) {}
```

Comment:
Move the `PythonByteContainer` class and `json_t` using declaration to its own file.